src/va_list.cr

```diff
@@ -17,4 +17,8 @@ struct VaList
       Intrinsics.va_end pointerof(ap)
     end
   end
+
+  @[Primitive(:va_arg)]
```

Comment:
This needs to be defined in `{% if compare_versions(Crystal::VERSION, "0.33.0-0") > 0 %}`

Otherwise, the std-lib will not build in 0.32.1 since the new primitive is not available yet.